ote_sdk/ote_sdk/entities/model.py

```diff
@@ -409,13 +413,27 @@ def model_size_reduction(self, value: float):
     @property
     def exportable_code(self) -> Optional[bytes]:
```

Comment:
Can this property return None now?

Reply:
Yes, looking at it again this implementation is wrong, both properties  I've made some changes, both properties should return None instead of raising errors.